### ISSUE TITLE
Conflicting method signatures for `store_node_certificate` depending on the type of storage

### ISSUE THREAD

**Issue by cygnusv** [nucypher/nucypher#661]
The signature of `store_node_certificate` is different depending on if the node storage is forgetful or file-based. In the latter case it requires a checksum address too, but in the former not. 

However, the following line of `Learner.remember_node()` is implicitly assuming that the node is forgetful (i.e., no checksum address is passed):
```
certificate_filepath = self.node_storage.store_node_certificate(certificate=node.certificate)
```

**Comment by KPrasch** [comment]
OK - This is addressed in `c0a9b3e` / PR #660

**Comment by KPrasch** [comment]
Fixed in PR #660
